applications/nrf_desktop/README.rst

```diff
@@ -741,35 +872,58 @@ Page Down key
     * |nRF_Desktop_cancel_operation|
 
 Dongle
-~~~~~~
+++++++
 
 The following predefined buttons are assigned to peer control operations for the dongle:
 
 SW1 button
+    * The SW1 button is located on the top of the dongle.
+
+      .. figure:: /images/nRF52840_dongle_front_callouts.svg
+         :scale: 25 %
+         :alt: nRF Desktop dongle - top view
+
+         nRF Desktop dongle - top view
+
     * Long-press to initialize peer erase, then double-press to confirm the operation.
       After the confirmation, all the Bluetooth bonds are removed.
     * Short-press to start scanning for both bonded and not bonded Bluetooth peripherals.
       The scan is interrupted if another peripheral connected to the dongle is in use.
     * |nRF_Desktop_cancel_operation|
 
+Debugging
+=========
 
-Power Management
-================
+Each of the nRF Desktop hardware reference designs has a slot for a dedicated debug board.
+See the following figures for the exact location of these slots:
 
-Reducing power consumption is important for every battery-powered device.
+.. tabs::
 
-The nRF Desktop peripherals are either suspended or powered off when they are not in use for a predefined amount of time:
+   .. group-tab:: Gaming mouse
```

Comment:
**Image edit:**
Please remove all captions _except_ the debug one from this image, and from the following two images.